### PR TITLE
New repo to create CDN.

### PR DESCRIPTION
**Pull request by Henry263**
Please refer this git repo for detail description.

https://github.com/Henry263/showError_n_IconJs

showError_n_iconJs

This repo is containing JS library where you can call function or attach method to element to appear the border around the element and icon next to the element. 

Purpose of showError_n_iconJs

For an example you have some HTML element that is showing an icon and border around the element for validation purpose or for any other purpose you can call the function or method that will apply automatically border and display icon for that element.

### CODE REVIEW

File: files/errornicon/info.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 author = "Harshil Patel"
 github = "https://github.com/Henry263/showError_n_IconJs"
 homepage = "https://github.com/Henry263/showError_n_IconJs/blob/master/README.md"
-mainfile = "showErrnIcon.min.js"
+mainfile = "errornicon.min.js"
 description = "This repo is containing JS library where you can call function or attach method to element to appear the border around the element and icon next to the element."
```
Comment (Henry263) [inline files/errornicon/info.ini]: Change the file name as per standard.

**Comment by Henry263** [comment]
@jsdelivrbot I have added the versioning to the project. Also just want to understand why these below 2 files are not proper?
Unexpected file, files/errornicon/errornicon.css
Unexpected file, files/errornicon/errornicon.min.js

Thanks for your help. Please give me more hint. I am new to this.

**Comment by megawac** [comment]
That error message should probably be improved, but what the bot is trying to tell you is to move errornicon.css and errornicon.min.js to a version folder. E.g.

files/errornicon/v1.0.0/errornicon.css

**Comment by Henry263** [comment]
@jsdelivrbot @megawac : I have updated the directory structure. Can you please take a look and let me know what else needed. Thanks for your help.

**Comment by megawac** [comment]
@Henry263 it looks like you did touch that file (appears that you added a new line). Can you remove that change and I'll merge the rest

**Comment by megawac** [comment]
If it's simpler for you, you can also just create a new pull request with the update.json and info.ini and the bot will take care of the rest

**Comment by Henry263** [comment]
@jsdelivrbot @megawac ; I have updated info and update.json file. Please do let me know is that correct or still I have to change something?
Thanks for your help.

**Pull request closed without merging**